packages.dhall

```diff
@@ -4,3 +4,29 @@ let upstream =
         sha256:60eee64b04ca0013fae3e02a69fc3b176105c6baa2f31865c67cd5f881a412fd
 
 in  upstream
+  with yoga-json.version = "v3.0.2"
```

Comment:
can bej dropped after https://github.com/purescript/package-sets/pull/1216 is released